### ISSUE TITLE
Update react to version 15.0

### ISSUE THREAD

**Issue by vslinko** [sapegin/react-styleguidist#116]
Today is the day when [first stable version of React is released](https://facebook.github.io/react/blog/2016/04/07/react-v15.html).
I hope that all popular instruments like this will support the new version soon.

But first we should wait for some of dependencies to update:

* [x] Wait for https://github.com/JedWatson/react-codemirror/issues/42 and update react-codemirror.
* [x] Wait for https://github.com/danmartinez101/babel-preset-react-hmre/pull/26 and update babel-preset-react-hmre.

**Comment by sapegin** [comment]
What doesn’t work now?

**Comment by vslinko** [comment]
I'm trying it right now and I'll share results soon.

**Comment by vslinko** [comment]
No issues found. Just warnings from npm:
```
npm WARN babel-preset-react-hmre@1.1.0 requires a peer of react@^0.14.1 but none was installed.
npm WARN react-codemirror@0.2.5 requires a peer of react@^0.14.0 but none was installed.
```

**Comment by sapegin** [comment]
We can update babel-preset-react-hmre right now, they don’t have React in peer dependencies anymore: https://github.com/danmartinez101/babel-preset-react-hmre/commit/73924d08da9d553b03505867357ac0d868563936

**Comment by JedWatson** [comment]
Just published an update to `react-codemirror` as well.

**Comment by sapegin** [comment]
@JedWatson Thanks!

**Comment by albertfdp** [comment]
@sapegin would it be possible to upgrade to [babel-preset-react-hmre](https://github.com/danmartinez101/babel-preset-react-hmre) 1.1.1? Current version requires it as a peer dependency and blocks the upgrade to 15 unfortunately

**Comment by sapegin** [comment]
@albertfdp Done in #128.
